platforms/corda-ent/charts/nmap/templates/deployment.yaml

```diff
@@ -298,7 +298,7 @@ spec:
         - name: nmap-logs
           mountPath: /opt/corda/logs
       imagePullSecrets:
-      - name: {{ .Values.image.imagepullsecret }}
+      - name: regcred
```

Comment:
Do not hard code, should be taken from value file

Reply:
refactored